submit/controllers/authorship.py

```diff
@@ -23,29 +26,72 @@
 Response = Tuple[Dict[str, Any], int, Dict[str, Any]]  # pylint: disable=C0103
 
 
+def _data_from_submission(params: MultiDict,
+                          submission: events.domain.Submission) -> MultiDict:
+    """Update form data based on the current state of the submission."""
+    if submission.submitter_is_author is not None:
+        params['authorship'] = (
+            AuthorshipForm.YES if submission.submitter_is_author
+            else AuthorshipForm.NO
```

Comment:
It now makes so much sense to me why you suggested changing how the validation options were written! Classy (in the English sense, not in the possible-Python-truthy-falsey sense).